### ISSUE TITLE
M0018 set cycle time test - configurable cycle length

### ISSUE THREAD

**Issue by pksoft72** [rsmp-nordic/rsmp_validator#462]
In this test

```ruby
    # 1. Verify connection
    # 2. Send control command to set cycle time
    # 3. Wait for status = true
    it 'M0018 set cycle time', sxl: '>=1.0.13' do |example|
      Validator::Site.connected do |task,supervisor,site|
        status = 5     ## <<<<<<<<< here
        plan = 0
        prepare task, site
        set_cycle_time status, plan
      end
    end

```

is hardcoded cycle length value to 5 seconds.

I prefer testing on near real conditions and our Cross RS4S controller supports only extending original plan and not shrinking.

- I can make some special very short plan to pass this test or 
- I can modify code to use configuration here with default value 5.

**Comment by emiltin** [comment]
I don't think we have any requirement that the  cycle time can be reduced. Perhaps we should change the test to first read the cycle time, then extend it, e.g. with 5s.

**Comment by pksoft72** [comment]
and then return back, otherwise it will grow with each test run

**Comment by emiltin** [comment]
You right, we should take care to reset it. Another option would be to keep the normal and extended cycle time as a test config, but I think it's easier if we can avoid adding test configs.

**Comment by emiltin** [comment]
The M0018 tests now extends the cycle time by 5s and reverts afterwards.
